### PR TITLE
Test that created loopback devices have expected size.

### PR DESCRIPTION
**Pull request by nfnt**

### CODE REVIEW

File: tests/test_dcos_e2e/docker_utils/test_loopback.py [hunk 1]
```diff
@@ -18,21 +18,34 @@ def test_loopback(self) -> None:
...
+                '-eq',
+                '"1048576"',
+                ']',
+            ]
+            exit_code, output = new_container.exec_run(
+                cmd=block_device_has_right_size,
+            )
             assert exit_code == 0, device.path + ': ' + output.decode()
 
+            new_container.stop()
```
Comment (adamtheturtle) [inline tests/test_dcos_e2e/docker_utils/test_loopback.py]: Let's move these back above the assertion. That means that even if the assertion fails, the container will be removed.

**Comment by adamtheturtle** [comment]
Thanks @nfnt - as CockroachDB was shipped on Friday to DC/OS OSS, our CI is broken, so let's be happy as long as this test and the linter pass.

File: tests/test_dcos_e2e/docker_utils/test_loopback.py [hunk 2]
```diff
@@ -18,21 +18,34 @@ def test_loopback(self) -> None:
         client = docker.from_env(version='auto')
 
         with DockerLoopbackVolume(size=1) as device:
```
Comment (adamtheturtle) [inline tests/test_dcos_e2e/docker_utils/test_loopback.py]: This makes me think - what do you think of changing this parameter name to be something like `size_megabytes`?

File: tests/test_dcos_e2e/docker_utils/test_loopback.py (hunk 1 above)
Comment (nfnt) [reply tests/test_dcos_e2e/docker_utils/test_loopback.py]: Good point! Though I'd rather use a `try... finally` block here, as there's more than one assertion that could fail.

File: tests/test_dcos_e2e/docker_utils/test_loopback.py (hunk 2 above)
Comment (nfnt) [reply tests/test_dcos_e2e/docker_utils/test_loopback.py]: The best solution would be a `Size` type, but I'm coming from statically typed languages, not sure if this is a common pattern in Python.

Comment (adamtheturtle) [reply tests/test_dcos_e2e/docker_utils/test_loopback.py]: The most common thing is to use something like `size_megabytes`. However, I have used [bitmath](https://pypi.org/project/bitmath/) to provide a type for sizes.

Comment (adamtheturtle) [reply tests/test_dcos_e2e/docker_utils/test_loopback.py]: I'll leave it to you to choose what to do, where doing nothing is an acceptable (to me) but not preferred choice. In a follow-up I hope so we can ship this.

File: tests/test_dcos_e2e/docker_utils/test_loopback.py (hunk 1 above)
Comment (adamtheturtle) [reply tests/test_dcos_e2e/docker_utils/test_loopback.py]: Either works for me.

File: tests/test_dcos_e2e/docker_utils/test_loopback.py [hunk 3]
```diff
@@ -18,21 +18,34 @@ def test_loopback(self) -> None:
...
+
+            block_device_exists_cmd = ['lsblk', device.path]
             exit_code, output = new_container.exec_run(
                 cmd=block_device_exists_cmd,
             )
-            new_container.stop()
-            new_container.remove()
+            assert exit_code == 0, device.path + ': ' + output.decode()
 
+            block_device_has_right_size = [
```
Comment (adamtheturtle) [inline tests/test_dcos_e2e/docker_utils/test_loopback.py]: Optional change:

Instead use a command which prints the device size, then make the comparison of something like `assert output == '1048576'`. That will help us find out what the size actually is if this comparison fails.

**Comment by adamtheturtle** [comment]
Thanks @nfnt ! I'm happy. Please make the change that means we are less likely to have leftover containers if something fails, then choose whether to make the other suggested changes / ignore them / do them in a separate PR.

File: tests/test_dcos_e2e/docker_utils/test_loopback.py (hunk 2 above)
Comment (nfnt) [reply tests/test_dcos_e2e/docker_utils/test_loopback.py]: `size_megabytes` it is. Will create an extra patch for that.

**Comment by adamtheturtle** [comment]
Thanks @nfnt this fixes https://jira.mesosphere.com/browse/DCOS_OSS-4359.

**Pull request merged**